extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -50,6 +50,10 @@ beforeEach(function (): void {
 	testContext = createContext();
 });
 
+afterEach(function (): void {
```

Comment:
Does this "afterEach" runs after each section of tests (differentiated by "describe") or after each individual tests?
A lot of comments won't make sense if it is for individual tests.

Reply:
This will run after each test in the block.

https://mochajs.org/#hooks

Although actually - this made me realize there's a bug here. Setting these at the global scope like this means that we'll run these two hooks for for EVERY test being ran - even ones outside this module. So these should actually all be scoped under a single describe in the file. 

This does bring up an interesting optimization we could potentially use where we have a single beforeEach specified in some base file that runs sinon.restore after every test. This would get rid of the need to do it for each module. It's not a huge improvement though and I kind of like being more explicit like this so will leave as is - but something to consider in the future if we want to do post-test cleanup on all tests.